src/common/runtime-bind.js

```diff
@@ -0,0 +1,23 @@
+define([
+	"./runtime-key"
+], function( runtimeKey ) {
+
+return function( args, cldr, fn, runtimeArgs ) {
+	var fnName = /^function\s+([\w\$]+)\s*\(/.exec( fn.toString() )[ 1 ],
+		locale = cldr.locale;
+
+	fn.runtimeKey = runtimeKey( fnName, locale, args );
+
+	// Stringify arguments.
+	args = JSON.stringify( args ).slice( 1, -1 );
```

Comment:
> Is `args` guaranteed to never contain a cyclical structure? Afaik that's the only reason for `JSON.stringify` to throw a `TypeError`

`args` is always formatters/parsers `arguments` after all type validations have been performed. It's not supposed to be cyclic. But, some arguments are potential candidates to be cyclic, for example if user passes an option with cyclic keys (despite the fact this would be really awkward) this is not verified.

Do you think it's worth handling that in advance, so JSON.stringify doesn't fail? Or it would be ok failing given the unexpected use case?